src/routes/Service/service.controller.ts

```diff
@@ -0,0 +1,50 @@
+import { AppDataSource } from './../../migration/data-source';
+import { Service } from './../../entity/Service';
+import { ResponseStatus } from './../../model/response-status';
+
+
+export const createService = async (req: any, res: any) => {
+    try {
+        const { serviceName, price, description, duration } = req.body;
+        if (
+            serviceName !== "" &&
+            price !== "" &&
+            description !== "" &&
+            duration !== ""
```

Comment:
This  validation can be done in the service entity check User entity and check how it handled in registerUser()